design/src/rfcs/rfc0022_refine_builder.md

```diff
@@ -0,0 +1,556 @@
+RFC: Evolving the new service builder API
+========================
+
+> Status: Accepted
```

Comment:
Is this accepted already? 😄

Reply:
It will if it gets merged, otherwise that doesn't matter 😛